packages/client/auth-core/src/components/AuthModal.tsx

```diff
@@ -89,17 +89,48 @@ export default function AuthModal({
 
     return (
         <ActionModal show={true} onClose={() => setModalOpen(false)}>
+            <div style={{ position: "relative", width: "100%" }}>
+                <button
+                    aria-label="Close"
+                    style={{
+                        width: "1.5rem", // equivalent to w-6
+                        position: "absolute",
+                        right: "1.5rem", // equivalent to right-6
+                        top: "1.5rem", // equivalent to top-6
+                        cursor: "pointer",
+                        color: "#67797F",
+                        outlineOffset: "4px",
+                        borderRadius: "100%",
+                    }}
+                    onClick={() => setModalOpen(false)}
+                >
+                    <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        width="24"
+                        height="24"
+                        viewBox="0 0 24 24"
+                        fill="none"
+                        stroke="currentColor"
+                        strokeWidth="2"
+                        strokeLinecap="round"
+                        strokeLinejoin="round"
+                    >
+                        <path d="M18 6 6 18" />
+                        <path d="m6 6 12 12" />
+                    </svg>
+                </button>
+            </div>
```

Comment:
Is it idiomatic to put this in it's own variable/component somewhere? If not it's fine!

Reply:
Added it in a separate component below 👍